### ISSUE TITLE
通知の基準

### ISSUE THREAD

**Issue by kikudesuyo** [kikudesuyo/price-comparison#12]
- 出力するタイミングを設定する
  - 最安値
  - 平均以下
  - etc..

**Comment by kikudesuyo** [comment]
また、通知された時のみグラフが表示されるようにする

**Comment by eteRnaL198** [comment]
10人とお見合いするときは､k=10/2.7≒3.7, k+1=4.7∴5人目が最適ってこと？笑

**Comment by kikudesuyo** [comment]
いや、そうではなくて、k=3.7だとしたら、k=3まではどんなに値が良くてもデータを切り捨てて、k=4以上で一番良かった値を採用するって感じだね。

e.g.)値が高いものを選ぶとする

```[10,13,12,12,11,14,10,15,19,20]```のリストがあったときに、順番に10、13、12は自動的に切り捨てて、その時点での最大値が13なので13以上の値が来た時に採用する。つまり今回であれば6番目にある14を採用することになる。
なのでこのリストのように後半に15,19,20の数値があったとしても採用することは出来ないけど、理論的には一番最適であることが言える。

**Comment by kikudesuyo** [comment]
csvファイルをcloud storageから呼び出せるようになってから着手
